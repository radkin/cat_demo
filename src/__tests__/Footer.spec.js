import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import Footer, { Counter, dataReducer } from '../Hamburger/Footer';

describe('Footer', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<Footer />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
