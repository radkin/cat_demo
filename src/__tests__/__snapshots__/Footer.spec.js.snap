// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot renders 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "color": "#de6432",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "center",
      "marginTop": "1rem",
      "position": "absolute",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "background": "#de6432",
        "height": "1px",
        "width": "90%",
      }
    }
  />
  <div
    style={
      Object {
        "padding": "1.5rem",
      }
    }
  >
    Skyline Labs Inc.
  </div>
</div>
`;
